packages/cli-kit/src/private/node/ui/components/Subdued.tsx

```diff
@@ -0,0 +1,16 @@
+import {Text} from 'ink'
+import chalk from 'chalk'
+import React from 'react'
+
+interface Props {
+  subdued: string
+}
+
+/**
+ * `Subdued` displays some text that represents a sub-process
+ */
+const Subdued: React.FC<Props> = ({subdued}: React.PropsWithChildren<Props>): JSX.Element => {
+  return <Text>{chalk.dim(subdued)}</Text>
```

Comment:
You can use the `Text` component property `dimColor` here so `<Text dimColor>{subdued}</Text>`

Reply:
+💯 @matteodepalo, thank you! I've applied the enhancement in a new commit :)